compositor_integration_tests/src/compositor_instance.rs

```diff
@@ -91,6 +91,7 @@ fn get_free_port() -> u16 {
 fn init_compositor_prerequisites() {
     static GLOBAL_PREREQUISITES_INITIALIZED: OnceLock<()> = OnceLock::new();
     GLOBAL_PREREQUISITES_INITIALIZED.get_or_init(|| {
+        env::set_var("LIVE_COMPOSITOR_NEVER_DROP_OUTPUT_FRAMES", "1");
```

Comment:
This should not be necessary if you lowered resolution. Is the issue still happening?

Reply:
I didn't test without it. But it seems to work without that env